BootloaderCorePkg/Tools/CommonUtility.py

```diff
@@ -47,6 +47,15 @@
             "SM3_256"     : 4,
     }
 
+# Hash values defined should match with cryptolib.h
+HASH_VAL_STRING = {
+            # Hash_Values : Hash_string
+             1    :  "SHA2_256",
+             2    :  "SHA2_384",
+             3    :  "SHA2_512",
+             4    :  "SM3_256",
+    }
+

```

Comment:
Use this instead:
HASH_VAL_STRING = dict((v, k) for k, v in HASH_TYPE_VALUE.iteritems())

Reply:
Thanks. Used "HASH_VAL_STRING = dict(map(reversed, HASH_TYPE_VALUE.items()))
" to work with PY 2.x and 3.x

Reply:
great!